.github/workflows/cd_push.yml

```diff
@@ -35,6 +35,15 @@ jobs:
           extra_tags: |
             ${{ inputs.image }}:${{ github.sha }}
 
+      # Run security checks on latest image built
+      - name: Dockle Security Check
+        uses: erzz/dockle-action@v1
+        with:
+          image: ${{ inputs.image }}:${{ github.sha }}
+          failure-threshold: fatal
+          exit-code: 1 # Stops pipeline if fatal issues found
+          accept-filenames: settings.py
```

Comment:
what does this do? The docs seem to mention something about credentials

Reply:
Docs are written poorly, checked the flag on the dockle command and it whitelists the files we don't wanna check.

```
Suspicious filename found : usr/local/lib/python3.8/site-packages/newrelic/api/settings.py (You can suppress it with "-af settings.py")
```
So by credentials filenames, it actually means `ignore the files with the filename X and don't complain about them having env variables set inside them`

Reply:
Ah, in that case can we maybe make this configurable from the project (we could keep this as default if nothing is passed I guess). I'm thinking each project might have different issues like this which we wouldn't want to hardcode here